msal/src/test/java/com/microsoft/identity/client/e2e/tests/network/AcquireTokenCIAMTest.java

```diff
@@ -56,27 +67,90 @@ public LabUserQuery getLabUserQuery() {
         return query;
     }
 
-    @Ignore //Disabling temporary due to tenant bug
+    public static AuthenticationCallback successfulVerifyIssuerCallback() {
+        return new AuthenticationCallback() {
+            @Override
+            public void onSuccess(IAuthenticationResult authenticationResult) {
+                Assert.assertFalse(StringUtil.isEmpty(authenticationResult.getAccessToken()));
+
+                Assert.assertNotNull(authenticationResult.getAccount().getClaims());
+                String idTokenIssuer = (String) authenticationResult.getAccount().getClaims().get("iss");
+                Assert.assertNotNull(idTokenIssuer);
+                Assert.assertTrue(idTokenIssuer.contains(CIAM_ISSUER));
```

Comment:
Line 78 is redundant. Line 79 essentially has to check nullability as well

Reply:
But the way you have it is probably more readable I guess. Fine either way

Reply:
Combining the two into one will produce an ambiguous result if it fails. Does it fail because of it's null, or because it doesn't contain CIAM_ISSUER string.